### PR TITLE
added post

### LINKED ISSUE #3: Add POST endpoint with url /api/users/toggleDriver
Implement after https://github.com/ucsb-cs156/proj-gauchoride/issues/6 and https://github.com/ucsb-cs156/proj-gauchoride/issues/7

Add POST endpoint with url /api/admin/users/toggleDriver that will toggle the driver field (i.e. switch it between true and false)

### PR DESCRIPTION
**Pull request by Anthonypella**
In this PR I added the post request in the backend to toggle the status of the driver. And I added tests.

### CODE REVIEW

**Review by seannoh — DISMISSED** [review]

**Review by Firoc — DISMISSED** [review]
All the tests cases pass. GJ adding the toggleDriver function and adding the test cases.

**Comment by andrewpengucsb** [comment]
Please add an appropriate description and a better title please. Also should not be committing changes to package-lock.json

File: frontend/package-lock.json
Comment (Anthonypella) [inline frontend/package-lock.json:1]: Reverted the modification to what it is on main (6.0) but it still includes the commit. should be fine.

**Comment by Anthonypella** [comment]
> Please add an appropriate description and a better title please. Also should not be committing changes to package-lock.json

Reverted the modification to what it is on main (6.0) but it still includes the commit. should be fine.

**Review by dingfan123 — APPROVED** [review]
LGTM

**Review by andrewpengucsb — APPROVED** [review]
LGTM

**Review by pconrad — COMMENTED** [review]
LGTM

**Pull request merged**